mapper/mapper.go

```diff
@@ -228,3 +230,19 @@ func getDifferenceOffset(period, interval string) int {
 	// only gets here if incomparable options are chosen in code
 	panic("unable to get difference offset from choosen period and interval values")
 }
+
+func hasFeaturedContent(featuredContent *[]model.Feature) bool {
+	if len(*featuredContent) > 0 {
```

Comment:
I like that we are separating out this functionality into its own function, wonder if we coluld just do:
```GO
return (len(*featuredContent) > 0)
```
As the function title gives enough context as to what is going on and this is a small function. (Not a requested change)

Reply:
A nice, small, refactor - have implemented it in the recent commit